content/20.pharmaceuticals.md

```diff
@@ -335,6 +334,7 @@ Therefore, despite the promising potential mechanism, initial results have not p
 
 CQ and hydroxychloroquine (HCQ) are lysosomotropic agents, meaning they are weak bases that can pass through the plasma membrane.
 Both drugs increase cellular pH by accumulating in their protonated form inside lysosomes [@doi:10.1186/1743-422X-8-163; @doi:10.1111/j.1529-8019.2007.00131.x].
+These drugs are used for the treatment and prophylaxis of malaria, as well as the treatment of lupus erythematosus and rheumatoid arthritis in adults [@url:https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=34496b43-05a2-45fb-a769-52b12e099341].<!--To Do: Check citation built-->
```

Comment:
```suggestion
These drugs are used for the treatment and prophylaxis of malaria, as well as the treatment of lupus erythematosus and rheumatoid arthritis in adults [@url:https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=34496b43-05a2-45fb-a769-52b12e099341].<!--To Do: Add manual reference-->
```
Does not work!

Reply:
I can go through and create the manual references after this is merged.  I'll search for comments with "manual reference" like you've been adding here.